src/sparsezoo/v2/objects/model_directory.py

```diff
@@ -60,13 +60,11 @@ def __init__(
     ):
 
         self.training: Directory = self._directory_from_files(
```

Comment:
How would you feel about something like this instead of manually calling `*_from_files()` for each of the expected files/directories? I think this could 1. make it a bit easier to quickly parse exactly what is expected to be in the directory, and 2. reduce chance for copy/paste error

```python
structure = {
    "sample_originals": Directory,
    "training": Directory,
    "sample_inputs": NumpyDirectory,
    "model_card": (File, "model.md"),
    ...
}

files = {}
for key, value in structure.items():
    if isinstance(value, tuple):
        ty, display_name = value
    else:
        ty, display_name = value, key
    
    if issubclass(ty, Directory):
        files[key] = self._directory_from_files(files, directory_class=ty, display_name=display_name)
    elif issubclass(ty, File):
        files[key] = self._file_from_files(files, display_name=display_name)
```

And then if files was just a list (or the dictionary key was exactly the display name), you could get rid of the tuple value and just go `{<display_name>: Type}`

Reply:
(And it makes the ones that don't fit into that structure a bit more obvious - I think just `sample_outputs`?)